packages/actor-http-fetch/lib/ActorHttpFetch.ts

```diff
@@ -88,12 +90,26 @@ export class ActorHttpFetch extends ActorHttp {
           (<Readable><any>response.body).destroy(error);
         };
       }
+
+      // We remove or update the timeout
+      if (requestTimeout !== undefined) {
+        const httpBodyTimeout = action.context?.get(KeysHttp.httpBodyTimeout) || false;
+        if (httpBodyTimeout && response.body) {
+          onTimeout = () => response.body?.cancel(new Error(`HTTP timeout of ${httpTimeout}ms enabled using the "httpTimeout" and "httpBodyTimeout" options`));
```

Comment:
I would suggest the following as error message:

```
HTTP timeout when reading the body of ${response.url}.
This error can be disabled by modifying the 'httpBodyTimeout' and/or 'httpTimeout' options.
```

Reply:
Done